src/dev/lrsql/user.clj

```diff
@@ -8,74 +8,6 @@
             [lrsql.util :as u]
             [lrsql.util.actor :as a-util]))
 
-;; H2
-(comment
```

Comment:
This comment block should probably stay but be converted to in-mem SQLite.

Reply:
58aec9caa963ca236b78da70600145396ef918d5

Reply:
Wrong commit link but still resolved